lib/media/ver10/profile/multicast_configuration.ex

```diff
@@ -15,7 +15,7 @@ defmodule Onvif.Media.Ver10.Profile.MulticastConfiguration do
 
     embeds_one :ip_address, IpAddress, primary_key: false, on_replace: :update do
       field(:type, Ecto.Enum, values: [ipv4: "IPv4", ipv6: "IPv6"])
-      field(:ipv4_address, :string)
+      field(:ipv4_address, :string, default: "0.0.0.0")
```

Comment:
`nil` was erroring out with invalid argument

Reply:
Important to note that some implementations also support the entire `tt:Address` == 0

>  (if this address is set to 0 no multicast streaming is enaled)

(typo by the docs)

Reply:
as a string or integer?

Reply:
as a integer